dialogy/plugins/text/classification/xlmr.py

```diff
@@ -42,8 +44,13 @@ def __init__(
         data_column: str = const.DATA,
         label_column: str = const.LABELS,
         state_column: str = const.STATE,
+        lang_column: str = const.LANG,
         args_map: Optional[Dict[str, Any]] = None,
         skip_labels: Optional[List[str]] = None,
+        prompts_map: Dict[Any, Any] = {},
+        use_prompt: bool = False,
```

Comment:
Why can't we make use of `use_state` here by assuming that `prompt` is part of the state?